checkbox-ng/checkbox_ng/urwid_ui.py

```diff
@@ -691,11 +691,11 @@ def interrupt_dialog(host):
         ('start', 'dark green,bold', 'black'),
     ]
     choices = [
-        _("Cancel the interruption and resume the session (ESC)"),
-        _("Kill test in progress and move on to next"),
-        _("Disconnect the master (Same as CTRL+C)"),
-        _("Stop the checkbox slave @{}".format(host)),
-        _("Abandon the session on the slave @{}".format(host)),
+        _("Nothing, continue testing (ESC)"),
+        _("Stop the test in progress and move on to the next"),
+        _("Disconnect but let the test session continue (CTRL+C)"),
+        _("Exit and stop the Checkbox service on the testbed at {}".format(host)),
+        _("Finalize this session and launch a new test plan"),
```

Comment:
"Finalize" may sound like the session will be wrapped up, and the user will have the opportunity to upload it to C3 or to download it. It's not the case: Checkbox removes the `incomplete` flag from the session `metadata` `flags` section, marking it as "finished" even though nothing has been done with it. By doing so, this session will not be resumable, but since there are probably jobs that have not been run, submitting it to C3 would not be very useful.

So I guess "Abandon" is more accurate.

Moreover, since we are talking about a session, it's better to stick to this word for the second part of the sentence, so you could write "and start a new one" for instance.

Reply:
Just one caveat to Abandon, the session is still there and downloadable for inspection, I wanted a verb to signal the fact that we are leaving it as finished and not removing it

Reply:
`"Stop the test in progress and move on to the next"`
One minor comment, I'd be just bit more specific here and say "test case" rather than "test"  as just "test" feels too generic, is it the test case, the whole test suite? a group of tests (e.g. the CPU tests)? But maybe that's just how I was reading it this morning.

`"Finalize this session and launch a new test plan"`
And with this one... what does "finalize" mean?  That wording always felt awkward to me... 

"End this test session, preserving its data, and launch a new test session" sounds more accurate to me, but that's just IMO.

I feel like these sorts of discussions always feel a bit like bike shedding, despite the fact that getting the words right is so very important, as @pieqq pointed out.